@xen-orchestra/lite/src/components/ui/UiKeyValueRow.vue

```diff
@@ -1,37 +1,32 @@
 <template>
-  <div class="ui-key-value-row">
-    <span class="key" v-if="$slots.key">
+  <tr class="ui-key-value-row">
+    <th v-if="$slots.key" class="key">
       <slot name="key" />
-    </span>
-    <span class="value">
+    </th>
+    <td :colspan="$slots.key ? 1 : 2" class="value">
       <slot name="value" />
-    </span>
-  </div>
+    </td>
+  </tr>
 </template>
 
 <script lang="ts" setup></script>
 
 <style lang="postcss" scoped>
-.ui-key-value-row {
-  display: flex;
-  gap: 1rem;
-  align-items: baseline;
-  margin: 0.5em 0;
+@import "@/assets/_responsive.pcss";
+
+.key,
+.value {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```

Comment:
`th` and `td` apply their own styles.
especially `th` with `font-weight: bold`.
```suggestion
  padding-bottom: 0.5rem;
  font-weight: 400;
```

Reply:
What do you mean they apply their own styles? You mean the browser defaults or a style that we applied somewhere else in the code? If you mean the browser defaults, they're inconsistent across browsers so it might be a good idea to (re)apply them here.

Reply:
Styles applied by the browser.